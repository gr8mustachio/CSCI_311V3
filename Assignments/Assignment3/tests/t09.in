findInMatrix
5 4 17
12 15 24 36
17 19 27 39
25 29 32 41
26 35 38 53
30 37 40 55
findInMatrix
5 4 53
12 15 24 36
17 19 27 39
25 29 32 41
26 35 38 53
30 37 40 55
findInMatrix
5 4 24
12 15 24 36
17 19 27 39
25 29 32 41
26 35 38 53
30 37 40 55
findInMatrix
5 4 30
12 15 24 36
17 19 27 39
25 29 32 41
26 35 38 53
30 37 40 55
findInMatrix
5 4 34
12 15 24 36
17 19 27 39
25 29 32 41
26 35 38 53
30 37 40 55
findInMatrix
5 4 36
12 15 24 36
17 19 27 39
25 29 32 41
26 35 38 53
30 37 40 55
findInMatrix
5 4 39
12 15 24 36
17 19 27 39
25 29 32 41
26 35 38 53
30 37 40 55
findInMatrix
5 4 12
12 15 24 36
17 19 27 39
25 29 32 41
26 35 38 53
30 37 40 55
findInMatrix
5 4 55
12 15 24 36
17 19 27 39
25 29 32 41
26 35 38 53
30 37 40 55
findInMatrix
5 4 18
12 15 24 36
17 19 27 39
25 29 32 41
26 35 38 53
30 37 40 55
findInMatrix
15 15 729
263	281	342	358	394	412	436	492	516	581	594	627	635	672	728
311	356	400	401	439	565	577	600	622	626	633	680	684	715	752
353	370	412	425	468	583	594	608	624	631	674	721	724	729	784
413	423	438	467	470	477	610	621	629	633	679	726	731	758	795
441	461	487	529	578	588	651	656	665	704	711	732	746	799	804
526	554	581	591	597	632	654	661	684	709	740	752	771	805	818
573	596	604	627	653	665	674	693	702	725	750	763	777	812	834
650	693	764	765	771	812	813	820	843	846	861	869	871	888	893
714	723	774	794	819	837	856	860	866	877	912	925	931	935	965
783	810	812	842	843	860	921	926	931	953	959	980	990	1018	1029
826	843	869	873	887	891	930	962	980	991	1003	1040	1057	1069	1070
831	852	879	884	899	904	946	980	1001	1014	1027	1068	1087	1107	1119
836	861	889	895	911	917	962	998	1022	1037	1051	1096	1117	1145	1168
841	870	899	906	923	930	978	1016	1043	1060	1075	1124	1147	1183	1217
846	879	909	917	935	943	994	1034	1064	1083	1099	1152	1177	1221	1266
findInMatrix
15 15 923
263	281	342	358	394	412	436	492	516	581	594	627	635	672	728
311	356	400	401	439	565	577	600	622	626	633	680	684	715	752
353	370	412	425	468	583	594	608	624	631	674	721	724	729	784
413	423	438	467	470	477	610	621	629	633	679	726	731	758	795
441	461	487	529	578	588	651	656	665	704	711	732	746	799	804
526	554	581	591	597	632	654	661	684	709	740	752	771	805	818
573	596	604	627	653	665	674	693	702	725	750	763	777	812	834
650	693	764	765	771	812	813	820	843	846	861	869	871	888	893
714	723	774	794	819	837	856	860	866	877	912	925	931	935	965
783	810	812	842	843	860	921	926	931	953	959	980	990	1018	1029
826	843	869	873	887	891	930	962	980	991	1003	1040	1057	1069	1070
831	852	879	884	899	904	946	980	1001	1014	1027	1068	1087	1107	1119
836	861	889	895	911	917	962	998	1022	1037	1051	1096	1117	1145	1168
841	870	899	906	923	930	978	1016	1043	1060	1075	1124	1147	1183	1217
846	879	909	917	935	943	994	1034	1064	1083	1099	1152	1177	1221	1266
findInMatrix
15 15 877
263	281	342	358	394	412	436	492	516	581	594	627	635	672	728
311	356	400	401	439	565	577	600	622	626	633	680	684	715	752
353	370	412	425	468	583	594	608	624	631	674	721	724	729	784
413	423	438	467	470	477	610	621	629	633	679	726	731	758	795
441	461	487	529	578	588	651	656	665	704	711	732	746	799	804
526	554	581	591	597	632	654	661	684	709	740	752	771	805	818
573	596	604	627	653	665	674	693	702	725	750	763	777	812	834
650	693	764	765	771	812	813	820	843	846	861	869	871	888	893
714	723	774	794	819	837	856	860	866	877	912	925	931	935	965
783	810	812	842	843	860	921	926	931	953	959	980	990	1018	1029
826	843	869	873	887	891	930	962	980	991	1003	1040	1057	1069	1070
831	852	879	884	899	904	946	980	1001	1014	1027	1068	1087	1107	1119
836	861	889	895	911	917	962	998	1022	1037	1051	1096	1117	1145	1168
841	870	899	906	923	930	978	1016	1043	1060	1075	1124	1147	1183	1217
846	879	909	917	935	943	994	1034	1064	1083	1099	1152	1177	1221	1266
findInMatrix
15 15 716
263	281	342	358	394	412	436	492	516	581	594	627	635	672	728
311	356	400	401	439	565	577	600	622	626	633	680	684	715	752
353	370	412	425	468	583	594	608	624	631	674	721	724	729	784
413	423	438	467	470	477	610	621	629	633	679	726	731	758	795
441	461	487	529	578	588	651	656	665	704	711	732	746	799	804
526	554	581	591	597	632	654	661	684	709	740	752	771	805	818
573	596	604	627	653	665	674	693	702	725	750	763	777	812	834
650	693	764	765	771	812	813	820	843	846	861	869	871	888	893
714	723	774	794	819	837	856	860	866	877	912	925	931	935	965
783	810	812	842	843	860	921	926	931	953	959	980	990	1018	1029
826	843	869	873	887	891	930	962	980	991	1003	1040	1057	1069	1070
831	852	879	884	899	904	946	980	1001	1014	1027	1068	1087	1107	1119
836	861	889	895	911	917	962	998	1022	1037	1051	1096	1117	1145	1168
841	870	899	906	923	930	978	1016	1043	1060	1075	1124	1147	1183	1217
846	879	909	917	935	943	994	1034	1064	1083	1099	1152	1177	1221	1266
findInMatrix
15 15 917
263	281	342	358	394	412	436	492	516	581	594	627	635	672	728
311	356	400	401	439	565	577	600	622	626	633	680	684	715	752
353	370	412	425	468	583	594	608	624	631	674	721	724	729	784
413	423	438	467	470	477	610	621	629	633	679	726	731	758	795
441	461	487	529	578	588	651	656	665	704	711	732	746	799	804
526	554	581	591	597	632	654	661	684	709	740	752	771	805	818
573	596	604	627	653	665	674	693	702	725	750	763	777	812	834
650	693	764	765	771	812	813	820	843	846	861	869	871	888	893
714	723	774	794	819	837	856	860	866	877	912	925	931	935	965
783	810	812	842	843	860	921	926	931	953	959	980	990	1018	1029
826	843	869	873	887	891	930	962	980	991	1003	1040	1057	1069	1070
831	852	879	884	899	904	946	980	1001	1014	1027	1068	1087	1107	1119
836	861	889	895	911	917	962	998	1022	1037	1051	1096	1117	1145	1168
841	870	899	906	923	930	978	1016	1043	1060	1075	1124	1147	1183	1217
846	879	909	917	935	943	994	1034	1064	1083	1099	1152	1177	1221	1266
findInMatrix
15 15 1119
263	281	342	358	394	412	436	492	516	581	594	627	635	672	728
311	356	400	401	439	565	577	600	622	626	633	680	684	715	752
353	370	412	425	468	583	594	608	624	631	674	721	724	729	784
413	423	438	467	470	477	610	621	629	633	679	726	731	758	795
441	461	487	529	578	588	651	656	665	704	711	732	746	799	804
526	554	581	591	597	632	654	661	684	709	740	752	771	805	818
573	596	604	627	653	665	674	693	702	725	750	763	777	812	834
650	693	764	765	771	812	813	820	843	846	861	869	871	888	893
714	723	774	794	819	837	856	860	866	877	912	925	931	935	965
783	810	812	842	843	860	921	926	931	953	959	980	990	1018	1029
826	843	869	873	887	891	930	962	980	991	1003	1040	1057	1069	1070
831	852	879	884	899	904	946	980	1001	1014	1027	1068	1087	1107	1119
836	861	889	895	911	917	962	998	1022	1037	1051	1096	1117	1145	1168
841	870	899	906	923	930	978	1016	1043	1060	1075	1124	1147	1183	1217
846	879	909	917	935	943	994	1034	1064	1083	1099	1152	1177	1221	1266
findInMatrix
15 15 1120
263	281	342	358	394	412	436	492	516	581	594	627	635	672	728
311	356	400	401	439	565	577	600	622	626	633	680	684	715	752
353	370	412	425	468	583	594	608	624	631	674	721	724	729	784
413	423	438	467	470	477	610	621	629	633	679	726	731	758	795
441	461	487	529	578	588	651	656	665	704	711	732	746	799	804
526	554	581	591	597	632	654	661	684	709	740	752	771	805	818
573	596	604	627	653	665	674	693	702	725	750	763	777	812	834
650	693	764	765	771	812	813	820	843	846	861	869	871	888	893
714	723	774	794	819	837	856	860	866	877	912	925	931	935	965
783	810	812	842	843	860	921	926	931	953	959	980	990	1018	1029
826	843	869	873	887	891	930	962	980	991	1003	1040	1057	1069	1070
831	852	879	884	899	904	946	980	1001	1014	1027	1068	1087	1107	1119
836	861	889	895	911	917	962	998	1022	1037	1051	1096	1117	1145	1168
841	870	899	906	923	930	978	1016	1043	1060	1075	1124	1147	1183	1217
846	879	909	917	935	943	994	1034	1064	1083	1099	1152	1177	1221	1266

