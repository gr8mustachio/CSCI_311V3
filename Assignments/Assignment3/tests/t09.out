17 is found in matrix M.
53 is found in matrix M.
24 is found in matrix M.
30 is found in matrix M.
34 is NOT found in matrix M.
36 is found in matrix M.
39 is found in matrix M.
12 is found in matrix M.
55 is found in matrix M.
18 is NOT found in matrix M.
729 is found in matrix M.
923 is found in matrix M.
877 is found in matrix M.
716 is NOT found in matrix M.
917 is found in matrix M.
1119 is found in matrix M.
1120 is NOT found in matrix M.
